resources/metadata/deployIntegrationArtifact.yaml

```
@@ -0,0 +1,79 @@
+metadata:
+  name: deployIntegrationArtifact
+  description: Deploy an CPI integration flow
+  longDescription: |
+    This function deploys an CPI Design time integration flow artifact, in to the integration runtime using OData API.
```

How about adding the description wanted in the documentation later here, instead of in the md file. As in:
```suggestion
  longDescription: |
    With this step you can deploy a integration flow artifact in to SAP Cloud Platform integration runtime using OData API.

    Learn more about the SAP Cloud Integration remote API for deploying an integration artifact [here](https://help.sap.com/viewer/368c481cd6954bdfa5d0435479fd4eaf/Cloud/en-US/08632076a1114bc1b6a1ecafef8f0178.html).
```